Assets/Scripts/Player/PlayerController.cs

```diff
@@ -215,6 +217,34 @@ public class PlayerController : MonoBehaviour {
 			else
 				UpdateCrouchColliders(crouchCollider, humanCollider, crouchGroundCollider, humanGroundCollider);
 		}
+		
+
+		if (isGrounded && !transformation.IsTransforming && 
+			transformation.State == TransformationState.Wolf && Mathf.Abs(xInput) > 0.1f) {
+			if (!wolfRunningSound.isPlaying && wolfRunningSound.time == 0) {
+				wolfRunningSound.Play();
+			}
+			else {
+				wolfRunningSound.UnPause();
+			}
```

Comment:
This seems to be able to be simplified if you set the sound to play on Awake and pause it instantly in Start. 
```suggestion
			wolfRunningSound.UnPause();
```